docs/source/index.mdx

```diff
@@ -4,19 +4,18 @@ title: Introduction to Apollo Server
 
 > 📣 **Apollo Server 4 is generally available!**
 >
-> [See what's new](./migration/)! 
+> [See what's new](./migration/)!
 >
 > Docs for Apollo Server 3 are [available here](/apollo-server/v3/).
 
 **Apollo Server is an [open-source](https://github.com/apollographql/apollo-server), spec-compliant GraphQL server** that's compatible with any GraphQL client, including [Apollo Client](/react). It's the best way to build a production-ready, self-documenting GraphQL API that can use data from any source.
 
-<img src="./images/frontend_backend_diagram.svg" width="500" alt="Diagram showing Apollo Server bridging frontend and backend" />
+<img src="./images/introduction.png" width="600" alt="Diagram showing Apollo Server bridging frontend and backend" />
 
 #### You can use Apollo Server as:
 
 * The GraphQL server for a [subgraph](./using-federation/apollo-subgraph-setup) in a federated supergraph
-* A [stand-alone GraphQL server](./api/standalone)
-* An add-on to your application's existing Node.js [middleware](./integrations/integration-index) (such as [Express](./api/express-middleware), AWS Lambda, or Fastify)
+* An add-on to any new or existing Node.js apps (such as [Express](./api/express-middleware), [MERN Stack](./integrations/mern.mdx), [AWS Lambda](https://www.npmjs.com/package/@as-integrations/aws-lambda), [Azure Functions](https://www.npmjs.com/package/@as-integrations/azure-functions), [Cloudflare](https://www.npmjs.com/package/@as-integrations/cloudflare-workers), [Fastify](https://www.npmjs.com/package/@as-integrations/fastify) and [more](./integrations/integration-index))
```

Comment:
I moved the placement, it's like that because I actually initially put it in integrations and someone gave feedback for tutorials. 

I definitely agree and although I want more tutorials, it's going to be a little bit until we get there. IMO, one day we have a tutorial for every integration.